openapi/build.sbt

```diff
@@ -3,13 +3,49 @@ import LocalCrossProject._
 
 val testsuite = LocalProject("testsuite")
 
+val `json-schema` =
+  crossProject.crossType(CrossType.Pure).in(file("json-schema"))
+    .settings(publishSettings ++ `scala 2.10 to 2.12`: _*)
+    .settings(
+      name := "endpoints-openapi-json-schema"
+    )
+
+val `json-schema-js` = `json-schema`.js
+val `json-schema-jvm` = `json-schema`.jvm
+
+val `json-schema-generic` =
+  crossProject.crossType(CrossType.Pure).in(file("json-schema-generic"))
+    .settings(publishSettings ++ `scala 2.10 to 2.12`: _*)
+    .settings(
+      name := "endpoints-openapi-json-schema-generic",
+      libraryDependencies += "com.chuusai" %% "shapeless" % "2.3.2"
+    )
+    .dependsOn(`json-schema`)
+
+val `json-schema-generic-js` = `json-schema-generic`.js
+val `json-schema-generic-jvm` = `json-schema-generic`.jvm
+
+val `json-schema-circe` =
+  crossProject.crossType(CrossType.Pure).in(file("json-schema-circe"))
+    .settings(publishSettings ++ `scala 2.10 to 2.12`: _*)
+    .settings(
+      name := "endpoints-openapi-json-schema-generic",
```

Comment:
The name duplicates `json-schema-generic`

Reply:
Good catch :)